test/unit/messaging/schemas/address.test.js

```diff
@@ -0,0 +1,84 @@
+const schema = require('../../../../app/messaging/schemas/address')
```

Comment:
for all of the schema, tests checking tht error is thrown and specifically Error is thrown need given (sommit tht I dnt think many of the validation in our other services do but they should, right? could be a tech debt ticket)

Reply:
The schema validation itself doesn't throw an error.  It more just creates an error property and then it's up to the application to check that result and decide if it should throw an error.

As these files are scoped to the schema, then I wouldn't expect to check errors are thrown.  But maybe we could check the contents of the error property more than we are.  Might be worth digging into that property definition a bit more as wouldn't want to hard code tests text too specifically and end up resulting in failures if Joi changes internally.  But agree we could do more.  Let me do some reading.

Reply:
Yup, agree on 1st point.

I _think_ what I was getting at was more around your `validate-statement` file and func which specifically runs the message against the schema, checks for any appended `.error` key and throws throw if so

Our other services do those steps fine but they usually do it w/in a larger func -- having this in its own means we can throw an error and catch it in those larger funcs.

I think my thinking was to adopt this way and implement across our other services, not sure?